carma/launch/guidance.launch

```diff
@@ -109,8 +109,10 @@
   <include file="$(find pure_pursuit_jerk_wrapper)/launch/pure_pursuit_jerk_wrapper.launch" />
 
     <!-- Route Following Plugin -->
-  <include file="$(find route_following_plugin)/launch/route_following_plugin.launch" />
-  
+  <group>
+    <include file="$(find route_following_plugin)/launch/route_following_plugin.launch" />
+    <remap from="upcoming_lane_change_status" to="$(optenv CARMA_GUIDE_NS)/upcoming_lane_change_status"/>
```

Comment:
As mentioned in the other PR having this remapping below the include line does nothing. If it is needed, move it above. Otherwise delete it